readme.md

```diff
@@ -5,7 +5,8 @@ _Diagram showing data flow relative to this project._
 
 ## Summary
 
-pycytominer-transform takes input data from CSV's, SQLite, or NPZ, for conversion to [Pycytominer](https://github.com/cytomining/pycytominer) relevant output in Parquet format.
+pycytominer-transform performs data transformations to assist cell biology image analysis for [Pycytominer](https://github.com/cytomining/pycytominer).
+It takes input data from CSV's, SQLite, or NPZ, for conversion to Pycytominer relevant output in Parquet format.
```

Comment:
```suggestion
```

I think this is now redundant after my suggestion above.

Reply:
Agreed, thank you for the suggestions!